_layouts/galatour.html

```diff
@@ -0,0 +1,163 @@
+---
+layout: default
+---
+<div id="tour-banner">
+    <div class="container">
+        <div class="row">
+            <div class="col-12 mb-4 col-md-6 col-lg-6 align-self-center">
+                <h1>2023春节游园会</h1>
+                <div><br>
+                    <p>
+                        时间：2023年1月21日（周六）， 13:00 -16:00 <br>
+                        地点： San Jose Center for the Performing Arts <br>
```

Comment:
所有涉及地址的地方，都最好能有一个 Google Maps 链接，方便大家直接导航。

```suggestion
                        地点： <a href="https://goo.gl/maps/3UJLFuwnhkT2QcMW7">San Jose Center for the Performing Arts</a> <br>
```